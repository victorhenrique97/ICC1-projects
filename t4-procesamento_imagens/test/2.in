2
img1.pgm
conv1.mat

