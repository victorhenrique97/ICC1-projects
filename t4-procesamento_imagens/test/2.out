P2
24 7
15
7 7 7 7 7 7 7 6 6 6 6 7 7 5 5 5 5 7 7 5 5 5 5 7
7 8 8 8 9 7 6 9 9 9 11 6 5 11 11 11 13 5 5 12 12 12 12 5
7 8 6 6 7 7 6 9 4 5 6 7 5 11 2 3 5 7 5 12 0 0 12 5
7 8 8 9 7 7 6 8 9 11 6 7 5 9 11 13 5 7 5 10 12 12 12 5
7 8 6 7 7 7 6 9 4 5 6 7 5 11 2 3 5 7 5 12 2 5 5 7
7 9 7 7 7 7 6 9 9 9 11 6 5 11 11 11 13 5 5 15 5 7 7 7
7 7 7 7 7 7 7 6 6 6 6 7 7 5 5 5 5 7 7 5 7 7 7 7
